src/Arbor/Api/Gateway/RestGateway.php

```diff
@@ -676,6 +676,7 @@ public function sendRequest($method, $url, array $options = [])
         switch ($code) {
             case 200:
             case 201:
+            case 204:
             case 422:
                 // Request succeeded or failed due to validation error(s), this is not an exception (return the response)
                 return $responsePayload;
```

Comment:
`return $responsePayload ?? [];` perhaps?

Reply:
Yeah, we could add that just to be safe. Thanks!